### PR TITLE
Cleanup old cython files before testing again

### PR DESCRIPTION
**Pull request by sb2nov**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`.
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

R: @robertwb PTAL

### CODE REVIEW

File: sdks/python/tox.ini [hunk 1]
```diff
@@ -48,6 +48,11 @@ deps =
 whitelist_externals=find
 commands =
   python --version
+  # Clean up all previous cython generated files.
+  - find apache_beam -type f -name '*.c' -delete
```
Comment (robertwb) [inline sdks/python/tox.ini]: What does the dash signify? I think I've gotten errors "bash cannot find command '-'" when running code like this.

Comment (sb2nov) [inline sdks/python/tox.ini]: dash ignores the command exit code so you still see the error but it doesn't fail the test.

Comment (robertwb) [inline sdks/python/tox.ini]: Hmm... Maybe I had an old version of tox.

**Comment by sb2nov** [comment]
Thanks.

**Pull request closed without merging**